### PR TITLE
Checkmates obv pandex

### PR DESCRIPTION
**Pull request by BrentWHR**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Changelog carboncastr
```
Comment (JobdePater) [inline NEWS.md:1]: soilptf

File: R/hydraulic_properties.R [hunk 2]
```diff
@@ -20,10 +20,12 @@ sptf_vg1 <- function(A_CLAY_MI,A_SILT_MI,A_SOM_LOI,B_DEPTH) {
   
   # Check input
   arg.length <- max(length(A_CLAY_MI), length(A_SILT_MI), length(A_SOM_LOI), length(B_DEPTH))
-  checkmate::assert_numeric(A_CLAY_MI, lower = 0, upper = 100, any.missing = FALSE, min.len = 1)
-  checkmate::assert_numeric(A_SILT_MI, lower = 0, upper = 100, any.missing = FALSE, min.len = 1)
-  checkmate::assert_numeric(A_SOM_LOI, lower = 0, upper = 100, any.missing = FALSE, min.len = 1)
-  checkmate::assert_numeric(B_DEPTH, lower = 0, upper = 2, len = arg.length)
+  check_numeric('A_CLAY_MI', A_CLAY_MI, FALSE, arg.length)
+  check_numeric('A_SILT_MI', A_SILT_MI, FALSE, arg.length)
+  checkmate::assert_numeric(A_SOM_LOI, lower = get_minval('A_SOM_LOI'),
```
Comment (JobdePater) [inline R/hydraulic_properties.R:25]: why not using check_numeric for A_SOM_LOI?

File: R/whc.R [hunk 3]
```diff
@@ -668,9 +671,11 @@ sptf_whc12 <- function(A_CLAY_MI, A_SAND_MI, D_BDS,
...
   # Check input
   arg.length <- max(length(A_CLAY_MI), length(A_SAND_MI), length(D_BDS))
-  checkmate::assert_numeric(A_CLAY_MI, lower = 0, upper = 100, len = arg.length)
-  checkmate::assert_numeric(A_SAND_MI, lower = 0, upper = 100, len = arg.length)
-  checkmate::assert_numeric(D_BDS, lower = 0, upper = 100, len = arg.length)
+  check_numeric('A_CLAY_MI', A_CLAY_MI, FALSE, arg.length)
+  check_numeric('A_SAND_MI', A_SAND_MI, FALSE, arg.length)
+  check_numeric('D_BDS', D_BDS, FALSE, arg.length)
+  checkmate::assert_subset(mp_fc, choices = c(10, 33), any.missing = FALSE)
+  checkmate::assert_numeric(mp_wp, any.missing = FALSE)
```
Comment (JobdePater) [inline R/whc.R:678]: are there no min or max values from literature?

Comment (BrentWHR) [reply R/whc.R:678]: Not sure about mp_wp, I'll add it to the todolist. The function description indicates mp_fc must be either 10 or 33.

**Review by JobdePater — APPROVED** [review]

**Pull request merged**